rabbitmq/check.py

```diff
@@ -293,6 +311,25 @@ def _get_metrics(self, data, object_type):
                     self.log.debug("Caught ValueError for %s %s = %s  with tags: %s" % (
                         METRIC_SUFFIX[object_type], attribute, value, tags))
 
+    def get_connections_stat(self, instance, base_url, object_type, vhosts, auth=None, ssl_verify=True, skip_proxy=False):
+        """
+        Collect metrics on currently open connection per vhost.
+        """
+        instance_proxy = self.get_instance_proxy(instance, base_url)
+        data = self._get_data(urlparse.urljoin(base_url, object_type), auth=auth,
+                              ssl_verify=ssl_verify, proxies=instance_proxy)
+
+        stats = defaultdict(int)
```

Comment:
nit: I don't think this needs to be a `defaultdict` anymore (a standard `dict`works)